lib/tofuenv-helpers.sh

```diff
@@ -122,18 +122,18 @@ export -f check_default_version;
 
 function cleanup() {
   log 'info' 'Performing cleanup';
-  local pwd="$(pwd)";
-  log 'debug' "Deleting ${pwd}/version";
+
+  log 'debug' "Deleting ${PWD}/version";
```

Comment:
where PWD is declared?

Reply:
@kvendingoldo `PWD` is declared/set by the `cd` builtin and by the shell on initialization. It's [in POSIX](https://pubs.opengroup.org/onlinepubs/9699919799.2018edition).